src/main/java/com/example/repository/AdministratorRepository.java

```diff
@@ -44,7 +44,7 @@ public class AdministratorRepository {
 	 * @throws EmptyDataAccessException 存在しない場合は例外を発生します
 	 */
 	public Administrator load(Integer id) {
-		String sql = "select id,name,mail_address,password from administrators where id=:id";
+			String sql = "select id,name,mail_address,password from administrators where id=:id";
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 24, expected level should be 4.